README.md

```diff
@@ -209,53 +225,55 @@ ssh -i ~/.ssh/connext-aws ubuntu@SERVER_IP bash indra/ops/collateralize.sh 1000
 
 A prod-mode indra hub exposes the following API ([source](https://github.com/ConnextProject/indra/blob/master/modules/proxy/prod.conf#L53)):
 
- - `/api/hub` is the prefix for the hub's api
- - `/api/hub/config` returns the hub's config for example
- - `/api/hub/subscribe` connects to the hub's websocket server for real-time exchange rate & gas price updates
- - `/api/eth` connects to the hub's eth provider
- - anything else, redirects the user to a daicard client
+- `/api/hub` is the prefix for the hub's api
+- `/api/hub/config` returns the hub's config for example
+- `/api/hub/subscribe` connects to the hub's websocket server for real-time exchange rate & gas price updates
+- `/api/eth` connects to the hub's eth provider
+- anything else, redirects the user to a daicard client
 
 ### ..from a [dai card](https://github.com/ConnextProject/card)
 
 Dai card in production runs a proxy with endpoints:
 
- - `/api/rinkeby/hub` -> `https://rinkeby.hub.connext.network/api/hub`
- - `/api/rinkeby/eth` -> `https://rinkeby.hub.connext.network/api/eth`
- - `/api/mainnet/hub` -> `https://hub.connext.network/api/hub`
- - `/api/mainnet/eth` -> `https://hub.connext.network/api/eth`
- - anything else: serves the daicard html/css/js files
+- `/api/rinkeby/hub` -> `https://rinkeby.hub.connext.network/api/hub`
+- `/api/rinkeby/eth` -> `https://rinkeby.hub.connext.network/api/eth`
+- `/api/mainnet/hub` -> `https://hub.connext.network/api/hub`
+- `/api/mainnet/eth` -> `https://hub.connext.network/api/eth`
+- anything else: serves the daicard html/css/js files
 
 ### Hub API
 
  1. AuthApiService
```

Comment:
Um, not relayed to the PR but these aren't correct there aren't POST and GET methods for these...